Cargo.toml

```diff
@@ -21,9 +21,19 @@ edition = "2018"
 members = ["rust", "wasm" ]
 
 [workspace.dependencies.snarkvm]
+path="../snarkVM"
```

Comment:
I'll update these when the associated snarkVM changes land: https://github.com/AleoHQ/snarkVM/pull/1367